Sources/TokamakCore/Styles/ButtonStyle.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Gene Z. Ragan on 07/22/2020.
```

Comment:
What’s the format for the header? Are we keeping these lines or removing them?

Reply:
I personally don't mind their presence (or absence) at all. They are added by Xcode, while files with no timestamps like that were created in other editors I guess (mine certainly were). I think adding these lines to files that don't currently have them is too much hassle, so if you want consistency it would be probably easier to just remove them everywhere.